### PR TITLE
Change vars to const

### PR DESCRIPTION
**Pull request by estensen**
## 📝 Summary

Extract some vars to const

## ⛱ Motivation and Context

const vals that shouldn't change

## ✅ I have run these commands

* [x] `make lint`
* [x] `make test-race`
* [x] `go mod tidy`

### CODE REVIEW

File: server/mock_relay.go [hunk 1]
```diff
@@ -17,9 +17,12 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
-var (
+const (
 	mockRelaySecretKeyHex = "0x4e343a647c5a5c44d76c2c58b63f02cdf3a9a0ec40f102ebc26363b4b1b95033"
 	// mockRelayPublicKeyHex = "0x8a1d7b8dd64e0aafe7ea7b6c95065c9364cf99d38470c12ee807d55f7de1529ad29ce2c422e0b65e3d5a05c02caca249"
```
Comment (ralexstokes) [inline server/mock_relay.go:22]: a bit off-topic but if we don't need this comment, can we just remove it?

Comment (estensen) [reply server/mock_relay.go]: Removed it

**Review by metachris — APPROVED** [review]

**Comment by metachris** [comment]
thanks 🙏

**Review by Seyvick — COMMENTED** [review]
.

**Pull request merged**